### DISCUSSION TITLE
DNS hostname resolving on Windows

### DISCUSSION THREAD

**Discussion by Lamera** [AdguardTeam/AdGuardHome#5101] [Q&A]
Hello

We recently moved from Microsofts DNS service to AdGuard home as a resolver for our internal network.
For Linux & Mac users all things are working flawlessly. But on Windows we have strange issues resolving hostnames of our servers since the change:
![grafik](https://user-images.githubusercontent.com/19840658/200030344-24e5709a-88ef-41ec-95b1-6d8d38493326.png)

`nslookup` can resolve the hostname, while `ping` or `ssh` can't. After investigation, we found out that it has something to do with the FQDN. When adding a "." after the hostname, for ex. intsolr1. we get an answer from AdGuard. Why is this and how can we tweak AdGuard home that it adds the domain part automatically?

**Comment by andekande** [comment]
Try this (192.168.178.1 is my router):

![grafik](https://user-images.githubusercontent.com/1636928/202289109-c59356e3-5f90-4ae7-8778-4185ef745b32.png)

![grafik](https://user-images.githubusercontent.com/1636928/202288865-42dd8809-53bc-4966-8dfe-100b219e0243.png)

**Reply by andekande** [reply]
reference #4636

**Reply by Lamera** [reply]
@andekande Thanks for your answer. But we have come up with another solution. We defined .local domains and added them in AGH. Then we added those domains as search suffix on the client machines. Your proposal still looks promising and worth a try.
